### PR TITLE
COM-29 Add lab results form

### PR DESCRIPTION
**Pull request by lauravignoli**

### CODE REVIEW

**Comment by tmvumbi2** [comment]
@lauravignoli the error below is displayed when loading the form; I can see the concept referred to in the list of concepts added, not sure what's causing the issue

image

**Comment by lauravignoli** [comment]
@tmvumbi2 Yes, I added a comment for that issue here https://github.com/jembi/cameroon-bahmni-config/pull/37, haven't managed to find why it's happening

**Comment by tmvumbi2** [comment]
@lauravignoli the date field isn't displayed when a result value is captured for `Epithelial cells per field`

image

**Comment by tmvumbi2** [comment]
@lauravignoli the date field isn't displayed for `Trichomonas vaginalis` as well 

image

**Comment by tmvumbi2** [comment]
@lauravignoli The date field isn't displayed for `Total cholesterol` and `Triglycerides` as well:

image

**Comment by tmvumbi2** [comment]
@lauravignoli same feedback for `HDL`

image

**Comment by tmvumbi2** [comment]
@lauravignoli the field `Platelets` should accept any text value according to the CSV (it's numeric with a unit and range on the form)

image

**Comment by tmvumbi2** [comment]
@lauravignoli `HIV Test` should be a section title according to the CSV, it's a Yes/No question on the form:

image

File: configuration/concepts/lab_results_form.csv [hunk 1]
```diff
@@ -0,0 +1,334 @@
...
+94659510-1ffc-4982-b3b7-9acee62dca84,,,Recherche des Micro Filaires Date de résultat,Recherche des Micro Filaires Test Date,Date de résultat,Test Date,Misc,Date,,
+baf5d5ac-47a0-4733-b58c-dfc8ef64530f,,,Skin snip,Skin snip,Skin snip,Skin snip,LabTest,Text,,
+4d4d0e97-af80-4e58-a63d-6739d133b4ec,,,Skin snip Date de résultat,Skin snip Test Date,Date de résultat,Test Date,Misc,Date,,
+7479640d-b96d-442c-be05-542bcd009a65,Crachants;Crachants Test Date;CSF parasitology;CSF parasitology Test Date;Coprology;Coprology Test Date;Rapid Diagnosis Malaria Test;Rapid Diagnosis Malaria Test Test Date;Malaria Test (GE);Malaria Test (GE) Test Date;Recherche des Micro Filaires;Recherche des Micro Filaires Test Date;Skin snip;Skin snip Test Date,,Département de Parasitologie,Parasitology Dept,Département de Parasitologie,Parasitology Dept,Misc,N/A,,
+3b5b8d72-2f86-48fc-9b81-0be908ed392c,Bacteriology Dept;Haematology Dept;Biochemistry Dept;Parasitology Dept,,Résultats de laboratoire - ajoutés manuellement,Lab results - add manually,Résultats de laboratoire - ajoutés manuellement,Lab results - add manually,Misc,N/A,,
```
Comment (tmvumbi2) [inline configuration/concepts/lab_results_form.csv]: @lauravignoli Please use full caps for this concept's name

**Review by tmvumbi2 — CHANGES_REQUESTED** [review]
Thanks @lauravignoli, the PR looks great overall. I left a few feedbacks

File: configuration/concepts/lab_results_form.csv (hunk 1 above)
Comment (lauravignoli) [reply configuration/concepts/lab_results_form.csv]: Fixed

**Comment by lauravignoli** [comment]
> @lauravignoli the error below is displayed when loading the form; I can see the concept referred to in the list of concepts added, not sure what's causing the issue
> 
> image

Fixed

**Comment by lauravignoli** [comment]
> @lauravignoli `HIV Test` should be a section title according to the CSV, it's a Yes/No question on the form:
> 
> image

Fixed

**Comment by lauravignoli** [comment]
@tmvumbi2 Fixed everything except the Total cholesterol and Tryglicerids, these are duplicated concepts, I'm busy finding a way to differenciate them in the form.

I used the existing concept for Platelets instead of creating a Text new one. Will check with Lisa.

**Review by tmvumbi2 — APPROVED** [review]

**Pull request merged**